### PR TITLE
Update home page

### PR DESCRIPTION
**Pull request by mec07**

### CODE REVIEW

File: src/Common/Button/Button.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import "./Button.scss";
 
 import cx from "classnames";
 
-function Button({ label, onClick, type, className, inline }) {
+function Button({ label, onClick, type, className, inline, social }) {
```
Comment (mec07) [inline src/Common/Button/Button.jsx]: what is the "social" thing for?

File: src/Team/TeamMember/TeamMember.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { Row, Col } from "react-grid-system";
+import Button from "../../Common/Button/Button";
+import Separator from "../../Common/Separator/Separator";
+
+import "./TeamMember.scss";
+
+export default function TeamMember({ name, title, bio, linkedin, photo }) {
+  console.log("bio = ", bio);
```
Comment (mec07) [inline src/Team/TeamMember/TeamMember.jsx]: worth removing this

File: src/Common/Button/Button.jsx (hunk 1 above)
Comment (dragosiordachioaia) [reply src/Common/Button/Button.jsx]: I wanted to do something else, then changed my mind..

File: src/Team/TeamMember/TeamMember.jsx (hunk 2 above)
Comment (dragosiordachioaia) [reply src/Team/TeamMember/TeamMember.jsx]: good catch

**Pull request merged**